.eslintrc.cjs

```diff
@@ -0,0 +1,66 @@
+module.exports = {
+  env: {
+    es2021: true,
+    node: true,
+  },
+  extends: [
+    'plugin:node/recommended',
+    'plugin:import/recommended',
+    'plugin:import/typescript',
+    'plugin:eslint-comments/recommended',
+    "plugin:@typescript-eslint/recommended",
+    'plugin:@typescript-eslint/recommended-requiring-type-checking',
+    'airbnb-base',
+    'airbnb-typescript/base',
+    'plugin:unicorn/recommended',
```

Comment:
This is an extensive collection of eslint rules that I collected over the years.

Eslint is a tool that helps to follow best practices in JavaScript development. A lot of rules fix the code automatically. The rules also have good documentation and examples, that you can find when they complain.

Some rules can be a bit annoying though, so you can add a `// eslint-disable-next-line rulename` (VsCode will suggest that for you if you mouseover the error). If you think the rule is stupid or broken, you can turn it off in this `.eslintrc.cjs` file. I kept all of them enabled at the start by it's ok if you want to disable some of them.